### PR TITLE
Notes disclaimer

### PR DESCRIPTION
**Pull request by littledan**
https://github.com/tc39/Reflector/issues/460

### CODE REVIEW

**Review by ljharb — APPROVED** [review]
Looks fine, but it seems to be equally applicable with the stenographer, or with all previous meetings where notes were taken.

**Comment by ljharb** [comment]
hmm, 539aaf0e is already pushed to main. i'll force push it out since this PR seems the same.

**Comment by ctcpip** [comment]
> including deleting comments which you do not wish to appear

perhaps there could be some clarified guidance on this item.  I have seen others delete comments that would be relevant to have in the notes.  could this perhaps be updated to language that limits this to one's own comments?  in practice of course, there is discretion in all these things, but maybe the existing language is too broad?

**Comment by littledan** [comment]
An alternative is that we could include the full text that I posted on the Reflector. Either way, Patrick from Ecma wants us to say something here because he imagines there is a GDPR compliance risk (though I am not so convinced).

**Comment by ctcpip** [comment]
Technically, GDPR does apply, and neither this disclaimer nor the text you posted on the Reflector brings us into GDPR compliance.  If the issue is GDPR compliance, we can attempt to solve for that, but again, this note doesn't move the needle at all on GDPR.

**Comment by ctcpip** [comment]
@tc39/chairs we need to discuss this

**Comment by littledan** [comment]
What are next steps on this PR?

**Comment by littledan** [comment]
It already lives in a Reflector issue. Landing the text here is a request of the Ecma Secretariat to serve GDPR concerns (possibly out of a misplaced excess of caution). But explaining notes is also a practical thing, and probably of interest to more people than the agenda rules always being copy-pasted.

I recommend the people who want to reject the Secretariat's advice to take it up with that group. If no one is going to do that, let's just land this patch; it really doesn't cause any harm.

File: template.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     1. Overview of communication tools
     1. Reminder to review Github Delegate teams (Jordan Harband)
 1. Find volunteers for note taking
+    1. Note: A detailed transcript of the meeting is being prepared and will be eventually posted on GitHub. You may edit this at any time during the meeting in Google Docs for accuracy, including deleting comments which you do not wish to appear. You may also request corrections or deletions after the fact, by editing the Google Doc (in the first two weeks after the TC39 meeting) or subsequently making a PR in the notes repository or contacting the TC39 chairs.
```
[the remark this answers is not in the document]
Comment (ctcpip) [reply template.md:67]: can we do this?  deleting is included in "editing" and resolves my concerns in a previous comment

**Review by ctcpip — CHANGES_REQUESTED** [review]

File: template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (littledan) [reply template.md:67]: We are specifically supposed to include deletion here.

**Comment by littledan** [comment]
@ljharb Yes, this is the intention, to announce at the beginning of the meeting alongside it being in written form here. Let me know if you have suggestions for how the written text should be edited.

**Comment by ljharb** [comment]
I think a short summary is fine; the actual announcement shouldn’t be inline.

File: template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ctcpip) [reply template.md:67]: I've updated the suggested to change; does that work?

**Review by ljharb — CHANGES_REQUESTED** [review]
being explicit that the content of this announcement doesn't belong on the agenda; a short summary would be fine. the content, if anywhere, belongs only on the reflector and/or on the notes docs, not here, since the agenda is for public consumption.

**Review by robpalme — APPROVED** [review]

**Comment by ljharb** [comment]
@ctcpip this doesn't resolve my concerns because the content is still on the agenda. using a `<details>` doesn't change that. PRs shouldn't ever be merged through blocks - I've pushed a commit that implements https://github.com/tc39/agendas/pull/1424#discussion_r1333602606; feel free to open a PR if that's problematic.

**Comment by ctcpip** [comment]
my apologies.  we discussed and merged and had yet to add a comment here re: best way to proceed.  in any case, I've reached out to Samina to get clarification on what their specific ask is (can we simply link to the text, etc), and next steps on wordsmithing.

**Pull request merged**